app/experimenter/nimbus-ui/src/components/PageHome/sortByStatus.ts

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { getStatus } from "../../lib/experiment";
+import { getAllExperiments_experiments } from "../../types/getAllExperiments";
+
+export interface ExperimentCollector {
+  draft: getAllExperiments_experiments[];
+  review: getAllExperiments_experiments[];
+  live: getAllExperiments_experiments[];
+  complete: getAllExperiments_experiments[];
+}
```

Comment:
Minor improvement, `Record<k,v>` type with a union key makes this dryer:

```tsx
export type ExperimentCollector = Record<
  "draft" | "review" | "live" | "complete",
  getAllExperiments_experiments[]
>;
```

Reply:
fancy